test/unit/helper-specs.js

```diff
@@ -22,38 +23,58 @@ describe('helpers', () => {
     });
   }));
 
-  describe('getAndroidPlatformAndPath', withMocks({fs, path}, (mocks) => {
+  describe('getAndroidPlatformAndPath', withMocks({fs}, (mocks) => {
     let oldAndroidHome;
-    beforeEach(function () {
+    let oldResolve;
+    before(function () {
       oldAndroidHome = process.env.ANDROID_HOME;
     });
-    afterEach(function () {
+    after(function () {
       process.env.ANDROID_HOME = oldAndroidHome;
     });
+    afterEach(function () {
+      if (oldResolve) {
+        path.resolve = oldResolve;
+      }
+    });
 
-    it('should return null if no ANDROID_HOME is set', async () => {
+    it('should return null if no ANDROID_HOME is set', async function () {
       delete process.env.ANDROID_HOME;
 
       await getAndroidPlatformAndPath().should.eventually.be.rejectedWith(/ANDROID_HOME environment variable was not exported/);
     });
-    it.skip('should get the latest available API', async () => {
+    it('should get the latest available API', async function () {
       process.env.ANDROID_HOME = '/path/to/android/home';
+
       mocks.fs.expects('exists')
         .exactly(2)
         .onCall(0).returns(false)
         .onCall(1).returns(true);
-      mocks.path.expects('resolve')
-        .exactly(3)
-        .onCall(0).returns('/path/to')
-        .onCall(1).returns('/path/to/apis1')
-        .onCall(2).returns('/path/to/apis2');
+      // manually mock `path`, to allow system to work
+      oldResolve = path.resolve;
```

Comment:
shouldn't this be set in beforeEach?

Reply:
It could be, but this is the only test that needs it.